src/renderer/components/Icon/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import classNames from 'classnames'
+
+export type IconName =
+  | 'arrow-left'
+  | 'bell'
+  | 'brightness-6'
+  | 'code-tags'
+  | 'cross'
+  | 'devices'
+  | 'favorite'
+  | 'forum'
+  | 'lead-pencil'
+  | 'list'
+  | 'open_in_new'
+  | 'person'
+  | 'swap_vert'
+
+type Props = {
+  className?: string
+  icon: IconName
+  size?: number
+}
+
+import styles from './styles.module.scss'
+
+export default function Icon({ size = 20, icon, className }: Props) {
+  return (
+    <span
+      className={classNames(styles.icon, className)}
+      style={{
+        WebkitMask: `url(../images/icons/${icon}.svg) no-repeat center`,
```

Comment:
also should specify the standard `mask` css property. we should use both, maybe tauri will need webkit mask, but could also be already so old that it does not matter, I did not closely follow it.

Reply:
you could also move `no-repeat` and `center` to the css file.